lib/mapper.js

```diff
@@ -47,6 +48,129 @@ exports.track = function(track){
 };
 
 /**
+ * Map Completed Order
+ *
+ * Docs: http://docs.klaviyo.com/article/60-ecommerce-integrating-a-custom-cart
+ *
+ * Must call Top level request for "Placed Order" and separate request for each item in products array
```

Comment:
This comment would be much improved if we described the structure of the object, see below